src/app/lib/providers/opensubtitles.js

```diff
@@ -47,19 +48,19 @@
         return this.fetch({
             imdbid: id
         }).then(function (data) {
-          App.vent.trigger('update:subtitles', data);
+            App.vent.trigger('update:subtitles', data);
```

Comment:
Could you just turn off you prettier? Otherwise, some small things that shouldn't be committed will be present on your PR.

Reply:
Agreed, my bad.

Reply:
@rafaell-lycan after careful review, I actually made this on purpose becase it was not aligned properly with the rest of the code before, so I made the changes to fix it in here. Take a look how it is now alligned with the rest of the code.